src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,22 @@
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0px 4px 6px rgba(128, 126, 126, 0.1);
+  .nav-list {
```

Comment:
Add a space before every element that you style, even when nested. Do it for styles below this as well